infinitest-runner/pom.xml

```diff
@@ -171,6 +171,11 @@
       <groupId>com.beust</groupId>
       <artifactId>jcommander</artifactId>
       <version>1.30</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
```

Comment:
@sardod do I need to add the entry also in the maven-dependency-plugin section of this pom.xml?

Reply:
The maven-dependency-plugin is used to unpack the dependencies for embedding in the runner jar? So I would say it depends if javax.annotation is required at runtime by runner.
What was the error that you solved by adding this dependency?

Reply:
javax.annotation is no more embedded with the jre since java 11.
Seems I need to add it then; i will do it soon.

Reply:
added